packages/wallets/src/utils/utils.ts

```diff
@@ -1,14 +1,85 @@
-import moment from 'moment';
+/**
+ * A utility function that returns a formatted date string after converting date objects/strings.
+ * @param dateInput - The date object, string, or Unix timestamp that needs to be converted/formatted.
+ * @param options - (Optional) Intl.DateTimeFormatOptions to specify the format of the returned date. Default format is 'YYYY-MM-DD' (ISO 8601).
+ * @param format - (Optional) The predefined format of the returned date. Available options: 'YYYY-MM-DD', 'DD MMM YYYY', 'MMM DD YYYY'.
+ * @param unix - (Optional) If true, treats the input as a Unix timestamp.
+ * @returns Formatted date string according to the specified format.
+ */
+export const getFormattedDateString = (
+    dateInput: Date | number | string,
+    options: Intl.DateTimeFormatOptions = { day: '2-digit', month: '2-digit', year: 'numeric' },
+    format = 'YYYY-MM-DD',
+    unix = false
+): string => {
```

Comment:
@lubega-deriv, why not mov this to the end, so you don't have to pass in the empty object everywhere  `getFormattedDateString` is called?
```suggestion
export const getFormattedDateString = (
    dateInput: Date | number | string,
    format = 'YYYY-MM-DD',
    unix = false,
    options?: Intl.DateTimeFormatOptions = { day: '2-digit', month: '2-digit', year: 'numeric' }
): string => {
```